src/widgets/Forms/AddExpenseForm/AddExpenseCalc.tsx

```diff
@@ -323,7 +342,7 @@ const AddExpenseCalculator = ({ closeModal }: TExpensesCalculatorProps) => {
           </Grid>
           <Grid xs={6} item>
             <Button
-              disabled={isPending || !amount || !selectedCategory || !selectedPaymentSource}
+              // disabled={isPending || !amount || !selectedCategory || !selectedPaymentSource}
```

Comment:
please make it disabled in case if we are creating/updating expense to prevent double click / miss click. all other LGTM